waiter/minimesosFile

```diff
@@ -0,0 +1,86 @@
+minimesos {
+    clusterName = "waiter"
+    loggingLevel = "INFO"
+    mapAgentSandboxVolume = false
+    mapPortsToHost = true
+    mesosVersion = "1.0.0"
+    timeout = 60
+
+    agent {
```

Comment:
I'm not sure if it's enough for all of them. I don't claim to have all integration tests passing with this PR :). Once we get this merged, I will focus on getting the integration tests passing in travis.